public/static/docs/get-started/import-data.md

```diff
@@ -68,7 +68,7 @@ outs:
 ```
 
 The `url` and `rev_lock` subfields under `repo` are used to save the origin and
-version of the dependency.
+[revision](https://git-scm.com/docs/revisions) of the dependency, respectively.
```

Comment:
should we have `rev` field as well here?

Reply:
The YAML above only has
```yaml
    repo:
      url: https://github.com/iterative/dataset-registry
      rev_lock: 7476a858f6200864b5755863c729bff41d0fb045
```
because no `--rev` was used in the `dvc import` example above (lines 26-7).

Reply:
so it uses to the default branch but doesn't save it in the DVC-file, it seems. Just the `rev_lock` commit hash.